charts/refractr/values.yaml

```diff
@@ -87,3 +95,81 @@ nginx-ingress:
   rbac:
     create: true
     scope: true
+
+
+# copied from https://github.com/bitnami/charts/blob/master/bitnami/nginx/values.yaml#L419-L493
+metrics:
+  enabled: true
+
+  ## Bitnami NGINX Prometheus Exporter image
+  ## ref: https://hub.docker.com/r/bitnami/nginx-exporter/tags/
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/nginx-exporter
+    tag: 0.8.0-debian-10-r18
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+  ## Prometheus exporter pods' annotation and labels
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+  ##
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9113"
```

Comment:
Should this be {{ .Values.metrics.service.port }} ?

Reply:
mmmmmm I'm not sure if that is a good choice actually. 
Choosing here what seems to be a random port number might not be the best option, but I believe Prometheus will autodiscover it.
The problem adding there the same port that used in the service definition is that probably those metrics can be fetched externally (in the internet) by hitting the endpoint URL/metrics.
I guess we need to test it